front/src/modules/tasks/hooks/useTasks.ts

```diff
@@ -11,7 +13,16 @@ export function useTasks() {
     TasksContext,
   );
 
-  const { data, loading } = useGetActivitiesQuery({
+  const { data: completeTasksData } = useGetActivitiesQuery({
```

Comment:
If we want refetch queries to work, we need to perform the two queries no matter on what tab we are. The other alternative would be to force a refetch when we switch tabs (adding a onClick on Tab, or using a useEffect in a <TabListHook> component). Not sure what you think it's the best way to handle this. I'm OK with doing two calls for now